XamarinCommunityToolkit/Behaviors/TagBehavior.shared.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text.RegularExpressions;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace Microsoft.Toolkit.Xamarin.Forms.Behaviors
+{
+    public class TagBehavior : BaseBehavior<Label>
+    {
+        string RegexPattern => $@"[{string.Join(string.Empty, TagTypes.Select(s => s.Symbol))}]\w+";
+
+        public static readonly BindableProperty CommandProperty =
+         BindableProperty.Create(nameof(Command), typeof(ICommand), typeof(TagBehavior), defaultBindingMode: BindingMode.TwoWay);
+
+        public static readonly BindableProperty TagTypesProperty =
+         BindableProperty.Create(nameof(TagTypes), typeof(IList<TagType>), typeof(TagBehavior), new List<TagType>(), defaultBindingMode: BindingMode.TwoWay);
```

Comment:
Why do these need ToWay bindings?

Reply:
Please review all bindable properties, I believe you just need to remove defaultBindingMode everywhere in this class.

Reply:
Fixed